src/queries/BB-BUS/getService.ts

```diff
@@ -0,0 +1,73 @@
+import gql from 'graphql-tag';
+
+export const GET_SERVICE = gql`
+  query getService($areaId: ID!, $id: ID!) {
```

Comment:
i do not know if this is the best place to put it, but i have some unwanted behavior.
i searched for a different city "brandenburg" and selected "brandenburg an der havel". then i searched for the pst "gewerbe abmeldung". when i opened the details, i reached the details of this pst but from "bad belzig" (initial area). i can see this in logs or in app when viewing "ansprechpartner".
logs at that point:
```
"organisationalUnits": Array [
    Object {
      "__typename": "OrganisationalUnit",
      "addresses": Array [
        Object {
          "__typename": "Address",
          "city": "Bad Belzig",
          "houseNumber": "6",
          "street": "Wiesenburger Straße",
          "zipcode": "14806",
        },
      ],
```

Reply:
this should be fixed now.